src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignerInfoTests.cs

```diff
@@ -676,7 +675,7 @@ public static void AddCounterSignerToUnsortedAttributeSignature()
         }
 
         [ConditionalFact(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
-        [SkipOnPlatform(PlatformSupport.MobileAppleCrypto, "DSA is not available")]
+        [SkipOnPlatform(PlatformSupport.AppleCrypto, "DSA is not available")]
```

Comment:
If at all possible, ConditionalFact.  But since it's already conditional on something else, maybe this is the best we can get?